packages/web-components/src/components/va-progress-bar/va-progress-bar.tsx

```diff
@@ -34,15 +47,15 @@ export class VaProgressBar {
         componentName: 'va-progress-bar',
         action: 'change',
         details: {
-          label: this.label || `${this.percent}% complete`,
+          label: this.label,
```

Comment:
Is the `@Watch` decorator what's making sure that this label is defined?

Reply:
I added the watch decorator to make sure the analytics receives the correct details if/when the percent prop is updated, but like you mentioned above, this won't work if a developer sets their own label because it'll override it.